### PR TITLE
feat: global availability block

### PR DESCRIPTION
**Pull request by fkakatie**
## Description

more dynamic map of global availability

- a SVG overlaid with the coordinates from a locations spreadsheet
- if you hover over a pin, it shows you the name of the city
- if you don’t hover, every 6 seconds a different city lights up
- source SVG is a 160 KB equirectangular borderless world map from [wikimedia](https://commons.wikimedia.org/wiki/Category:SVG_maps_of_the_world_with_equirectangular_projection#/media/File:Equirectangular_projection_world_map_without_borders.svg) 

## How Has This Been Tested?

https://avail-block--helix-website--adobe.aem.page/drafts/fkakatie/global

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: blocks/availability/availability.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+function generateRandomNumber(max, excluding) {
+  //  minimum is inclusive, maximum is exclusive
+  let randomIndex;
+  do {
+    randomIndex = Math.floor(Math.random() * Math.floor(max));
+  } while (randomIndex === excluding);
+  return randomIndex;
+}
+
+function tooltipAfterOverride(tooltip, axis, pixels) {
```
Comment (kptdobe) [inline blocks/availability/availability.js:12]: This method contains a lot of maths, but it is not obvious what it is doing or why it is needed...

File: blocks/availability/availability.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
+import { createTag } from '../../scripts/scripts.js';
+
+function generateRandomNumber(max, excluding) {
+  //  minimum is inclusive, maximum is exclusive
```
Comment (kptdobe) [inline blocks/availability/availability.js:4]: No `minimum`... ? What is `excluding` ?

**Review by kptdobe — APPROVED** [review]
I approved because code looks good (at least visual result is great). But you should write a little bit of JS doc / comments because some parts look tricky and deserve some explanation. Think about someone else having to fix this code.

**Review by kptdobe — APPROVED** [review]
Thanks, much easier to understand now!

**Review by trieloff — APPROVED** [review]

**Pull request merged**